tests/unit/launcher/test_ds_arguments.py

```diff
@@ -98,3 +99,8 @@ def test_core_deepscale_arguments():
     assert hasattr(args, 'deepspeed_config')
     assert type(args.deepspeed_config) == str
     assert args.deepspeed_config == 'foo.json'
+
+
+def test_core_binding_arguments():
```

Comment:
Could we add some negative tests to ensure we catch invalid core list specifications?

Reply:
Will add some negative tests.

Reply:
Negative cases added as in the following link:
https://github.com/microsoft/DeepSpeed/pull/2881/files#diff-d616664f976bb37f412dc2bd3380c255241b97008eb0ed90dc685f43bd7488aaR108